components/common/listingCart/singleItemCart/ListingCartSingleItemCart.vue

```diff
@@ -82,9 +82,11 @@ watch(
 )
 
 watch(
-  () => item.value.listPrice,
+  () => item.value?.listPrice,
   (value) => {
-    fixedPrice.value = Number(value)
+    if (value) {
+      fixedPrice.value = Number(value)
```

Comment:
why type conversion?
listPrice is already a  number

![image](https://github.com/kodadot/nft-gallery/assets/22791238/2a00ba40-06ce-49dc-a024-3799114594f6)

Reply:
Got it! 😄